cmd/tikv-ctl/src/cmd.rs

```diff
@@ -604,10 +603,22 @@ pub enum RaftCmd {
         // `regions` must be None when `all_regions` is present,
         // so we left `all_regions` unused.
         #[allow(dead_code)]
-        #[structopt(long, required_unless = "regions", conflicts_with = "regions")]
+        #[structopt(long, conflicts_with = "regions")]
         /// Print info for all regions
         all_regions: bool,
 
+        #[structopt(long, default_value = "")]
+        /// hex start key
+        start: String,
+
+        #[structopt(long, default_value = "")]
+        /// hex end key
+        end: String,
+
+        #[structopt(long, default_value = "0", max_values = 10240)]
```

Comment:
What about setting the max value to 10000? 10240 sounds weird 😂.

Reply:
![sUavxRWr1a](https://user-images.githubusercontent.com/53859786/202071482-4b68f217-2764-4aa5-a6b1-5b8a9506ab8a.jpg)
is to synchronize the pd-ctl